scripts/src/main/java/com/mathewsachin/fategrandautomata/scripts/entrypoints/AutoBattle.kt

```diff
@@ -28,7 +28,8 @@ fun IFgoAutomataApi.isInSupport(): Boolean {
 fun IFgoAutomataApi.isInventoryFull() =
     // We only have images for JP, NA and KR
     prefs.gameServer in listOf(GameServerEnum.En, GameServerEnum.Jp, GameServerEnum.Kr)
-            && images[Images.InventoryFull] in locations.inventoryFullRegion
+            && locations.inventoryFullEnhancementButtonRegion.exists(images[Images.InventoryFull], similarity = 0.23)
```

Comment:
23 % similarity is incredibly low. I think you had to set it that low because you used `&&` instead of `||` in line 32.

Can you try again with the changes from my commit?